### PR TITLE
Z columns doc to have different template.

### PR DESCRIPTION
**Pull request by navinrathore**
Zingg internal columns (Z columns) should have different template.

### CODE REVIEW

File: core/src/main/java/zingg/documenter/ColumnDocumenter.java [hunk 1]
```diff
@@ -86,4 +92,24 @@ public Map<String, Object> addStopWords(Dataset<Row> data, String fieldName, Map
 		
 		return params;
 	}
+
+	public String getColumnBaseContent (String col) {
+		String message = "The field '" + col + "' is internally used by Zingg.";
+
+		if (col.equals(ColName.CLUSTER_COLUMN)) {
```
Comment (sonalgoyal) [inline core/src/main/java/zingg/documenter/ColumnDocumenter.java:99]: This needs to go in the template, not in the java code

File: core/src/main/java/zingg/documenter/ColumnDocumenter.java [hunk 2]
```diff
@@ -26,6 +26,7 @@ public class ColumnDocumenter extends DocumenterBase {
 
 	private final String CSV_TEMPLATE = "stopWordsCSV.ftlh";
```
Comment (sonalgoyal) [inline core/src/main/java/zingg/documenter/ColumnDocumenter.java:27]: One thing I seem to have missed in my earlier reviews - we should have column templates which include stop word templates. Then it much easier to keep stuff modular and extend etc as needed

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: core/src/main/java/zingg/documenter/ColumnDocumenter.java (hunk 2 above)
Comment (navinrathore) [reply core/src/main/java/zingg/documenter/ColumnDocumenter.java]: moved content in a separate template and included in the parent template

File: core/src/main/java/zingg/documenter/ColumnDocumenter.java (hunk 1 above)
Comment (navinrathore) [reply core/src/main/java/zingg/documenter/ColumnDocumenter.java]: Moved to Template in if else blocks.

**Comment by navinrathore** [comment]
#256

**Review by sonalgoyal — CHANGES_REQUESTED** [review]
a. z col list should just be columns that star with z_
b. StopWordDocumenter should be introduced and included in the ColumnDocumenter. All code and logic for StopWords should go in there. We can then add more documenters for columns. eg data quality. 
c.

**Comment by navinrathore** [comment]
> a. z col list should just be columns that star with z_ b. StopWordDocumenter should be introduced and included in the ColumnDocumenter. All code and logic for StopWords should go in there. We can then add more documents for columns. eg data quality.

b. Moved the StopWords stuff in a separate class
a.  z col list is indeed formed from 6 z_ columns. Is there any issue in it?

**Comment by sonalgoyal** [comment]
It will be neater to have a check in the pattern for z columns instead of keeping a list

**Comment by sonalgoyal** [comment]
Please also rebase this pr as there is a merge conflict

**Comment by navinrathore** [comment]
> It will be neater to have a check in the pattern for z columns instead of keeping a list

We go for documentation with marked data as input.  It includes field definition fields and only few z_columns. DONT_USE fields also excluded.
But, Documenter is generating docs for all fields (excludes DON'T USE) but includes all z_fields. As of now, only ones part of marked data are shown with hyperlink heading. 
There was some issue working with sequence of columns of marked data. then we moved to get information from FieldDefinition and z Col list.

**Comment by navinrathore** [comment]
> Please also rebase this pr as there is a merge conflict

resolved conflict.

**Pull request merged**